### PR TITLE
:whale: Refactor docker compose

### PR DESCRIPTION
**Pull request by jefftriplett**
tl;dr Modern Compose drops the `docker-` prefix and I'd like to make the project easier for our incoming new devs. 

I need to test in production to see if I can update `fabric.py` to the latest docker version when I have time to put out fires.

### CODE REVIEW

**Comment by ryancheley** [comment]
I did a cursory review of it and it looks good. i'll need to download it and run it locally to see if I run into any issues (which will most likely be Friday or Saturday). 

@jefftriplett In terms of testing on production, if you need any help with trying to put out fires, let me know when you're thinking you'll be able to try it out and I can potentially be there with you if you're looking for an extra set of hands/eyes 😄

**Comment by jefftriplett** [comment]
@ryancheley that'd be great. I'll probably do test it Friday if I don't get to it sooner.

File: docs/docs/install.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ You'll want to make sure your local environment is ready by installing the follo
 
 ### Docker
 
-If you don't have them installed yet, install [Docker] and [docker-compose].
+If you don't have them installed yet, install [Docker] and [Compose].
```
Comment (saadmk11) [inline docs/docs/install.md:13]: Should we mention the minimum required versions here?

File: docs/docs/install.md [hunk 2]
```diff
@@ -74,7 +74,7 @@ And then login into the admin interface (/admin/) and create a profile for your
 We use a **Mock** system of creating sample data in our tests and for running a development version of the site. To create some development data, just run:
 
 ```shell
-docker-compose run --rm django python manage.py load_dev_data
+docker compose run --rm django python -m manage load_dev_data
```
Comment (saadmk11) [inline docs/docs/install.md:77]: :100:

**Review by saadmk11 — COMMENTED** [review]
I ran this branch on GitHub Codespaces and it is working as expected. :+1:

**Comment by ryancheley** [comment]
> I ran this branch on GitHub Codespaces and it is working as expected. 👍

I need to remember that GitHub Codespaces are a thing!

**Comment by jefftriplett** [comment]
> > I ran this branch on GitHub Codespaces and it is working as expected. 👍
> 
> I need to remember that GitHub Codespaces are a thing!

I need to remember this too. I eyeballed the change but Codespaces might be worth mentioning in our docs for our new djangonaut space recruits.

File: docs/docs/install.md (hunk 1 above)
Comment (jefftriplett) [reply docs/docs/install.md:13]: Great idea. I don't know what the minimum should be.

**Pull request merged**